src/main/java/com/fvv/bookstore/dao/ConnectionFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.fvv.bookstore.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * Class Connection Factory to implement a connection with the database.
+ * 
+ * Created on 03/21/2018
+ * @author Fatima Vasquez
+ * @version 1.0
+ */
+public class ConnectionFactory {
+
+	private static final String DB_DRIVER = "com.mysql.jdbc.Driver";
+	private static final String DB_URL = "jdbc:mysql://localhost:3306/bookstore_db";
+	private static final String DB_USER = "root";
+	private static final String DB_PASS = "test1234";
```

Comment:
It is not a good idea to keep user/password in the code. For now its okay, but before ending the project you need to hide this.

Reply:
Ok!

Reply:
we will deal with this later, proceed with the project